proto/proto.bzl

```diff
@@ -220,6 +220,7 @@ def _rust_proto_compile(protos, descriptor_sets, imports, crate_name, ctx, is_gr
 
     return rustc_compile_action(
         ctx = ctx,
+        attr = ctx.attr,
```

Comment:
Yeah changing this to only needed attributes is a huge change, and I agree that it should be done in a separate PR, not in this one. :+1: